test/utils/test_draw.py

```diff
@@ -5,7 +5,70 @@
 
 from kornia.testing import assert_close
 from kornia.utils import create_meshgrid, draw_convex_polygon, draw_rectangle
-from kornia.utils.draw import draw_line
+from kornia.utils.draw import draw_line, draw_point2d
+
+
+class TestDrawPoint:
+    """Test drawing individual pixels."""
+
+    def test_draw_point2d_rgb(self, dtype, device):
+        """Test plotting multiple [x, y] points."""
+        points = torch.tensor([(1, 3), (2, 4)], device=device)
+        color = torch.tensor([5, 10, 15], dtype=dtype, device=device)
+        img = torch.zeros(3, 8, 8, dtype=dtype, device=device)
+        img = draw_point2d(img, points, color)
+        for x, y in points:
+            assert_close(img[:, y, x], color.to(img.dtype))
+
+    def test_draw_point2d_grayscale_third_order(self, dtype, device):
+        """Test plotting multiple [x, y] points on a (1, m, n) image."""
+        points = torch.tensor([(1, 3), (2, 4)], device=device)
+        color = torch.tensor([100], dtype=dtype, device=device)
+        img = torch.zeros(1, 8, 8, dtype=dtype, device=device)
+        img = draw_point2d(img, points, color)
+        for x, y in points:
+            assert_close(img[:, y, x], color.to(img.dtype))
+
+    def test_draw_point2d_grayscale_second_order(self, dtype, device):
+        """Test plotting multiple [x, y] points on a (m, n) image."""
+        points = torch.tensor([(1, 3), (2, 4)], device=device)
+        color = torch.tensor([100], dtype=dtype, device=device)
+        img = torch.zeros(8, 8, dtype=dtype, device=device)
+        img = draw_point2d(img, points, color)
+        for x, y in points:
+            assert_close(torch.unsqueeze(img[y, x], dim=0), color.to(img.dtype))
+
+    def test_draw_point2d_with_mismatched_dims(self, dtype, device):
```

Comment:
Check the [CONTRIBUTING.md](https://github.com/kornia/kornia/blob/master/CONTRIBUTING.md) to see how to do the exception tests